packages/zosjobs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
 
+## Recent Changes 
+- Enhancement: Added new functions to support the changing of a job class and the hold status of a job. Can now call `zowe jobs modify job [jobid]` with options `--jobclass`, `--hold` and `--release`. [#1156](https://github.com/zowe/zowe-cli/issues/1156)
```

Comment:
For the SDK changelog, we should mention new methods that were added (like `ModifyJobs.modifyJob`), rather than CLI commands which are only applicable to the "@zowe/cli" package.